Sources/StreamChatUI/ChatMessageList/RETHINK/MessageContentView.swift

```diff
@@ -98,17 +100,26 @@ class MessageContentView<ExtraData: ExtraDataTypes>: _View, UIConfigProvider {
         layoutOptions = options
         
         var constraintsToActivate: [NSLayoutConstraint] = []
+        var fullWidthViews: [UIView] = []
         
         // Main container
         mainContainer.alignment = .axisTrailing
```

Comment:
I actually think we can achieve the full-width functionality just by changing this to `.fill`. It probably won't work yet but it should eventually :)

Reply:
The `fill` requires us to rework existing components so they can stretch but still look right